ingest/build.gradle

```diff
@@ -6,7 +6,7 @@ plugins {
 dependencies {
 	testCompile group: "org.springframework.boot", name: "spring-boot-starter-test", version: "2.1.5.RELEASE"
 	compile group: "org.json", name: "json", version: "20180813"
-	implementation("com.connexta.ingest:ingest-api-rest-spring-stubs:0.1.0-SNAPSHOT")
+	implementation("com.connexta.ingest:ingest-api-rest-spring-stubs:0.1.1-SNAPSHOT")
```

Comment:
❗️ I'm running the release for 0.1.1 right now, so we should wait until that passes and drop the `-SNAPSHOT`